.github/workflows/cicd.yml

```
@@ -157,4 +157,4 @@ jobs:
           git push origin release/${{ github.run_id }}
 
       - name: 릴리즈 PR 생성
-        run: gh pr create --base main --head release/${{ github.run_id }} --title "🚀 $(jq -r .pull_request.title \"$GITHUB_EVENT_PATH\") -> main"
+        run: gh pr create --base main --head release/${{ github.run_id }} --title "🚀 $(jq -r .pull_request.title \"$GITHUB_EVENT_PATH\") -> main"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)